lib/discourse_livestream/chat_channel_extension.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module DiscourseLivestream
+  module ChatChannelExtension
+    extend ActiveSupport::Concern
+
+    prepended do
+      has_one :topic_chat_channel,
```

Comment:
this is a bit generic, could it be `livestream_topic_chat_channel`?

Reply:
let me test, I copied from what we have from topic: https://github.com/discourse/discourse-livestream/blob/eb5eab9869d6dd807a3d2e3dd856c2d3757c48a6/lib/discourse_livestream/topic_extension.rb#L8-L9

Reply:
using this way gives me this error:
<img width="778" alt="Screenshot 2024-08-16 at 15 53 11" src="https://github.com/user-attachments/assets/1299e39a-ddac-4586-bca2-f997b1ebb07c">

Reply:
did you remove the `class_name` key? Keep that there as you had it